release-notes/release-notes58.md

```diff
@@ -25,7 +25,9 @@ keywords: releasenotes bugfixes
 
 The following new features were added for the {{site.productname}} 5.8 release.
 
-### Feature name
+### Improved keyboard navigation for the inline boundary feature
+
+Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys, see: [Content appearance options - Inline boundaries]({{site.baseurl}}/configure/content-appearance/#inline_boundaries).
```

Comment:
Need to explain why we are linking to that other page. Since this functionality isn't documented there, we nee to tell them why we have the link. We link there so we don't have to re-explain inline boundaries here in the release notes. Unfortunately, we don't explain what we mean by "inline boundaries" anywhere in the docs atm.

For example:
```suggestion
Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys.

For information on inline boundaries, see: [Content appearance options - Inline boundaries]({{site.baseurl}}/configure/content-appearance/#inline_boundaries).
```